README.md

```diff
@@ -2,7 +2,7 @@
 
 This document contains the instructions for using and developing Edge.
 
-The full Edge documentation is [part of our Device Management documentation site](https://developer.pelion.com/docs/device-management-edge/latest/introduction/index.html), where you can also find the [API documentation](https://developer.pelion.com/docs/device-management-edge/latest/edge-api-references/index.html). For comments or questions about the documentation, please [email us](mailto:support@mbed.org).
+The full Edge documentation is [part of our Device Management documentation site](https://developer.pelion.com/docs/device-management-edge/latest/introduction/index.html), where you can also find the [API documentation](https://developer.pelion.com/docs/device-management-edge/latest/edge-api-references/index.html). For comments or questions about the documentation, please [email us](mailto:support@izumanetworks.com).
```

Comment:
The links that are currently Pelion will be fixed later on?

Reply:
Yes. Lots of links fixing once we get all the web stuff set up.